apps/application-system/api/src/environments/environment.ts

```diff
@@ -18,10 +31,11 @@ const devConfig = {
       process.env.WEB_FRONTEND_PORT ?? '4242'
     }/umsoknir`,
     emailOptions: {
-      useTestAccount: !(process.env.USE_SES === 'true'),
-      useNodemailerApp: process.env.USE_NODEMAILER_APP === 'true' ?? false,
+      useTestAccount: !(process.env.USE_SES === 'true') ?? '',
+      useNodemailerApp:
+        process.env.USE_NODEMAILER_APP === 'true' ?? false ?? '',
```

Comment:
Guðjón is also changing those lines on PR https://github.com/island-is/island.is/pull/15514 maybe the two of you should make sure the two changes won't clash an break things

Reply:
Noted, will keep an eye on this :)